.storybook/preview.css

```diff
@@ -16,6 +16,9 @@
   .sbdocs.sbdocs-toc--custom {
     width: 15rem;
     min-width: 15rem;
```

Comment:
Note: these styles only affect the documentation site, not the shipped Optics CSS